### ISSUE TITLE
About the data

### ISSUE THREAD

**Issue by mrzhangzizhen123** [mikevoets/jama16-retina-replication#9]
Hello, professor
   I recently trying to reappear the your code, I met a problem, but before I kaggle downloaded DR data set, contains five seven training set and test set, but I put these data sets in eyepacs folder after running eyepacs. Sh extract only the training set, after running eyepacs. Sh is only a pool folder, inside a folder called "train" and a lot of pictures, but the train is empty inside.I only see one pool file when I run eyepacs.sh. Can you help me solve it?

**Comment by mikevoets** [comment]
Hi. Can you please send the exact command you used to run `eyepacs.sh`? Also, what platform are you using?

**Comment by mrzhangzizhen123** [comment]
非常感谢你的回复，代码已经跑通了，结果和你的差不多，但是ROC曲线图画不出来，如果我在你的代码上换一下网络，比如Resnet、inceptipn-resnet等，请给一些建议。谢谢

**Comment by mrzhangzizhen123** [comment]
Thank you very much for your reply, the code has been running and the result is similar to yours, but the ROC curve drawing is not available. If I change the network on your code, such as Resnet, inceptipn-resnet, please give me some Suggestions.thank you.

**Comment by mikevoets** [comment]
The code does not generate a ROC curve drawing. You can save the specifity/sensitivity operating points to a file with the `-so` parameter, for example: `python evaluate.py -so operating_points.csv`. Then you can use the `operating_points.csv` file to draw a ROC curve.
